src/lib/data-cache.ts

```diff
@@ -243,6 +244,29 @@ export async function cacheManyContentUrlsWithProgress(
                         }
                     }
                 }
+
+                if (hasInlineMedia) {
+                    const [data] = await responseClone.json();
```

Comment:
We may want to iterate through the array instead of getting just the first item here. Since for some content we have multiple items in the array (e.g. FIA content).

Reply:
I considered this and decided it should be treated as one document with inline media. The media is required if you want the document. But I will check in with the product team.